2.adoc

```diff
@@ -72,5 +72,173 @@ Optional description
 . Minus A
 . Minus B
 
+=== Typora
+
+Dekstopový (Windows, macOS, Linux) komerčný markdown-only (GitHub flavored) editor.
+Dokument je renderovaný v tom istom okne hneď počas písania.
+Pri pohybe kurzoru medzi jednotlivými blokmi sa vyrenderovaný blok rozpadne na zdrojový kód, ktorý je možné editovať.
+link:+https://typora.io+[https://typora.io]
+
+==== Porovnání rozdílů
+* Jedná sa o desktopový editor, nie webovú komponentu. 
+* Podporuje iba jazyk markdown (github-flavored). 
+* Kvôli chýbajúcej command pallete vyžaduje používanie myši ak si užívateľ nepamätá klávesovú skratku. 
+* Podporuje zmenu vzhľadu pomocou css šablón.
+
+
+==== Zhodnocení UI
+
+===== Defaultné rozhranie so zobrazenou outline dokumentu:
+
+image::https://i.imgur.com/n3vyN3i.png[]
+
+Okno editora je rozdelené na 2 hlavné časti, v postrannom paneli naľavo je zobrazená outline dokumentu alebo súborová štruktúra.
+Vpravo je samotný dokument. Informácia o počte slov sa nachádza v title bare. Pod záhlavím okna sa nachádza panel s otvorenými tabmi. 
+
+
+===== Focus mode
+Celý dokument vybledne, zvýraznený je len aktívny blok
+
+image::https://i.imgur.com/u6NGu0Z.png[]
+
+===== typewriter mode
+Zaručí aby aktívny riadok bol vždy v strede editoru. 
+
+===== Source code mode
+Namiesto priamo live renderingu je stále zobrazený zdrojový kód, ktorý je zvýraznený nie len farebne, ale aj veľkosťou písma, odsadením (vlastne vyrenderovaný dokument, kde je miesto textu stále markdown).
+Pri zapnutí tohto módu nefunguje drag n drop na pridanie obrázku do dokumentu
+
+image::https://i.imgur.com/NWJ2Mtz.png[]
+
+===== Outline 
+* podporu zbalenia/rozbalenia je nutné zapnúť v nastaveniach
+* Pevná šírka 
+
+===== Iné
+* Podpora viacerých tabov
+* Chýbajú presety (napr. vim-like, emacs-like) klavesových skratiek 
+* Nápoveda je formou markdown dokumentu 
+
+==== Pozorování UI v kontextu markup-editor
+* Chýba command paleta
+  ** Alternatívou ku klávesovým skratkám je klasické menu, kde je užívateľ nutený siahnúť po myši
+  ** klávesové skratky na styling dokumentu (nadpisy, odsadenie, tabulky, bloky kódu, citacie, zoznamy, foot notes, horizontal lines, bold, underline,..., vloženie obrázku) (neaktívne v source code mode)
+* Nepodporuje režim so zobrazeným zdrojákom a vyrenderovaným dokumentom vedľa seba
+  ** štandardne live rendering v tom istom okne (preview)
```

Comment:
tohle bych asi nenazýval *preview*, ale spíš WYSIWYG editor

Reply:
Ak ťa niekto podpori tak kludne premenujem (osobne mi to je jedno). Terminologiu som prevzal z https://typora.io/